WcaOnRails/config/locales/en.yml

```diff
@@ -380,7 +390,6 @@ en:
     errors:
       not_found: "not found"
       already_assigned: "already assigned to a different user"
-      required: "required"
```

Comment:
I removed this since I had to use it from different controllers, and because we can use the common `simple_form.required.text`.